3p/nameframe.max.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <script>
+        window.onload = function() {
```

Comment:
I wondered that myself!  The issue seems to be that if you do a `document.write()` _during_ load, it only appends to the doc.  If you wait until load is completed, it replaces it, which is the desired behavior here.